craco.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require(`path`);
```

Comment:
/nit You could use single quotes as this string is not being interpolated, but it's a matter of code style.